packages/react/src/components/pickers/TagPicker/TagItem.styles.ts

```diff
@@ -103,6 +104,11 @@ export function getStyles(props: ITagItemStyleProps): ITagItemStyles {
           ? `${effects.roundedCorner2} 0 0 ${effects.roundedCorner2}`
           : `0 ${effects.roundedCorner2} ${effects.roundedCorner2} 0`,
         selectors: {
+          [`.${classNames.isSelected} &`]: {
+            // selected but not focused, assume color and bg of parent
+            color: 'inherit',
+            background: 'transparent',
```

Comment:
I'd kinda lean towards making this the same as `:focus`, or just using both selectors for the same style block.